src/utils/EnemyDefense.js

```diff
@@ -4,30 +4,30 @@
  * @param {Object} enemyBuild The enemy taking the damage of the character
  * @returns The DEF multiplier
  */
-function EnemyDamageReduction(characterBuild, enemyBuild) {
+function DMGReductionByDefense(characterBuild, enemyBuild) {
   const def = 5 * enemyBuild.enemy_level + 500;
   const defDmgReduction =
     def / (def + 5 * characterBuild.character_level + 500);
 
-  return defDmgReduction;
+  return 1 - defDmgReduction;
```

Comment:
`defDmgReduction` is the amount the enemy is blocking, so `1-defDmgReduction` is the amount getting through
also for consistency with the resistance function, which returns the percent to multiply dmg with